### ISSUE TITLE
[Feature] Score-giving mechanism for Readiness

### ISSUE THREAD

**Issue by michellzappa** [envisioning/tdb#17]
We need an algorithm for averaging the score given by multiple Users for the technology.

Example:
- [Defense Drones] has Q1: 3, Q2: 4, Q3: 2, etc.
- User Michell reviews [Defense Drones] for [Q3] giving it [4 points] on [2015-06-09] with optional remark _"Amazon just announced their delivery service"_

All information in the above example is important and should be used to estimate the given average for all questions.

In other words, score = equation using above information to triangulate what several experts think about that technology's readiness, measured as a function of time.

**Comment by NDuma** [comment]
@michellzappa we discussed possibilities for this type of issue over skype several weeks ago; and, had planned a follow up call...

The data also needs to be weighted by individual user rankings; and, there are strong ways to go about this.

As well as noting a 'decay' factor for techs which have not been 'touched' / updated in some [factor of] time.

Multiple user inputs on each information type to be 'averaged' in their respective data types per tech prior to calculating the aggregate output.

As from Readiness.io [current]:
[User] votingStrength={User Experience}*{Community Weighting}
[Date Stamp]
[Technology Reviewed]
Q1 = Viability // averageSum = ([1User = value"4"], [2User = value"7"], [nUser = value"n"])
Q2 = Demonstration
Q3 = Breakthrough
Q4 = Investment
Q5 = Cost
Q6 = Explainability
Q7 = Usefulness
Q8 = Necessity
Q9 = Competition
Q10 = Relevance

Harmonic Sum (Q1...Q10) = techReadinessIndexScore

This structure is *highly* simplified -- Let me know if you'd like to discuss confidence intervals & probabilistic backbone I had mentioned previously; as, it's already built.
